### ISSUE TITLE
Error Building Dockerfile 3.5.2 

### ISSUE THREAD

**Issue by dunhampa** [rocker-org/shiny#60]
I am getting the following error when trying to build shiny-verse Dockerfile 3.5.2 Dockerfile

The downloaded source packages are in
	‘/tmp/downloaded_packages’
Error: installation of package ‘sf’ had non-zero exit status
In addition: Warning messages:
1: In install.packages(pkgs, ...) :
  installation of package ‘units’ had non-zero exit status
2: In install.packages(pkgs, ...) :
  installation of package ‘rgdal’ had non-zero exit status
3: In install.packages(pkgs, ...) :
  installation of package ‘rgeos’ had non-zero exit status
4: In install.packages(pkgs, ...) :

**Comment by cboettig** [comment]
@dunhampa Are you sure you're building [shiny-verse 3.5.2 Dockerfile](https://github.com/rocker-org/shiny/blob/master/shiny-verse/Dockerfile)?  Your error message suggests that some spatial data packages are failing to install, and, well, those aren't included in that Dockerfile so it looks like you're building something else...

Note that rocker/shiny-verse:3.5.2 is built nightly on the Docker Hub already: https://hub.docker.com/r/rocker/shiny-verse , so if the build was broken we'd probably see an error there too.  (Also means you can always `docker pull rocker/shiny-verse:3.5.2`, you shouldn't have to build it locally; though of course that should work too).

**Comment by dunhampa** [comment]
@cboettig Apologies! Thank you for the reply.

I was seeing the error when I building a modified Dockerfile (adding the leaflet package).   As you pointed out this is not the shiny-verse 3.5.2 Dockerfile. I did just confirm I could build shiny-verse 3.5.2 Dockerfile. This was a mistake on my end. 

Thanks for the additional notes as well -- I was a building as a learning exercise so appreciate the additional context. 

Closing.
